### PR TITLE
[packaging] Add 2023.04.02-7 release notes to .metainfo.xml

### PR DESCRIPTION
**Pull request by Ferenc-**

### CODE REVIEW

File: packaging/app.organicmaps.desktop.metainfo.xml [hunk 1]
```diff
@@ -113,6 +113,29 @@
...
+     <description>
+       <p>Highlights:</p>
+         <ul>
+           <li>New OpenStreetMap data as of March 29</li>
+           <li></li>
+           <li></li>
+         </ul>
+       <p>Map Styles:</p>
+         <ul>
+           <li>Many new shop types</li>
```
Comment (biodranik) [inline packaging/app.organicmaps.desktop.metainfo.xml:126]: • New OpenStreetMaps data as of March 29
• Correctly import KML bookmarks with track data
• Allow adding music and musical_instruments in the Editor
• Sewing shops and haberdasheries are searchable
• Distinguish supermarkets and convenience stores
• New shop types: agrarian, appliance, bed, outpost, gas, key cutter, locksmith, carpet, motorcycle repair, pet grooming, hi-fi, bathroom furnishing, curtain, fashion accessories, cannabis, herbalist
• Added money transfer, bicycle charging stations, power plant
• Made captions mandatory for capitals with a star icon
• Made park, forest, and scrub colors different
• Added metal construction icons
• Improved aerial ways, cable cars, power lines, fences
• Reduce Barcelona subway visibility
• Fixed isolines that are drawn over buildings, roads, etc.
• Fixed some POI captions displayed earlier than their icons
• Improved German, Turkish, Belarusian, Hungarian translations
• Changed displayed name to Organic Maps
• OM version is selectable in About

Comment (Ferenc-) [reply packaging/app.organicmaps.desktop.metainfo.xml]: Done

File: packaging/app.organicmaps.desktop.metainfo.xml [hunk 2]
```diff
@@ -113,6 +113,39 @@
...
+           <li>Improved aerial ways, cable cars, power lines, fences</li>
+           <li>Reduce Barcelona subway visibility</li>
+           <li>Fixed isolines that are drawn over buildings, roads, etc.</li>
+           <li>Fixed some POI captions displayed earlier than their icons</li>
+         </ul>
+       <p>Translations:</p>
+         <ul>
+           <li>Improved German, Turkish, Belarusian, Hungarian translations</li>
+         </ul>
+       <p>Linux:</p>
```
Comment (biodranik) [inline packaging/app.organicmaps.desktop.metainfo.xml:142]: Ditto

**Review by biodranik — APPROVED** [review]

File: packaging/app.organicmaps.desktop.metainfo.xml (hunk 2 above)
Comment (Ferenc-) [reply packaging/app.organicmaps.desktop.metainfo.xml]: I have dropped this paragraph and moved the items into the Highlights paragraphs.

**Review by biodranik — APPROVED** [review]
Thank you!

**Pull request merged**